examples/riscv/README.md

```diff
@@ -0,0 +1,34 @@
+# Prerequisites for RISC-V examples 
+
+## Intalling the RISC-V cross-compilation toolchain
+
+Clone the RISC-V GNU Compiler Toolchain:
+
+```shell
+$ git clone https://github.com/riscv/riscv-gnu-toolchain
+```
+
+Install the prerequisites, e.g., on Ubuntu:
+
+```shell
+$ sudo apt-get install autoconf automake autotools-dev curl python3 python3-pip libmpc-dev libmpfr-dev libgmp-dev gawk build-essential bison flex texinfo gperf libtool patchutils bc zlib1g-dev libexpat-dev ninja-build git cmake libglib2.0-dev
+```
```

Comment:
I understand usage of `$` to distinguish between the system shell and other command prompt interfaces, however here it is used only for the two first commands and nowhere else. In this particular README, there's also no other prompt to confuse the system shell with. I suggest to remove them, unless a good reason says otherwise.

Reply:
Done in 26617a7